go.mod

```diff
@@ -3,13 +3,18 @@ module github.com/cockroachdb/cockroach-go/v2
 go 1.13
 
 require (
-	github.com/jackc/pgx/v4 v4.6.0
-	github.com/jinzhu/gorm v1.9.12
+	github.com/jackc/pgx v3.6.2+incompatible // indirect
```

Comment:
do you know why pgx v3 is added here? i don't think it should be

Reply:
yea, I agree. I've removed it from go.mod.